nixos/modules/services/web-apps/meme-bingo-web.nix

```
@@ -22,7 +22,7 @@ in {
 
       baseUrl = mkOption {
         description = mdDoc ''
-          URL to be used for the HTML <base> element on all HTML routes.
+          URL to be used for the HTML \<base> element on all HTML routes.
```

```suggestion
          URL to be used for the HTML \<base\> element on all HTML routes.
```

A quick `rg "\\\<"` shows [other](https://github.com/NixOS/nixpkgs/blob/b58b0b0311f083da10dbb936539fe1cb85c1a6fb/nixos/modules/services/networking/ndppd.nix#L111) `mdDoc`s that also escape `<text>` like this.